app/assets/stylesheets/darkswarm/_shop-filters.css.scss

```diff
@@ -107,26 +107,37 @@
 // Alert when search, taxon, filter is triggered
 
 .alert-box.search-alert {
-  background-color: $clr-yellow-light;
-  border-color: $clr-yellow-light;
+  background-color: $white;
   color: #777;
-  font-size: 0.75rem;
-  padding: 0.5rem 0.75rem;
+  font-size: 1em;
+  padding: 0.35em 0 0;
+  border: 0;
+  margin: 0;
 
-  span.applied-properties {
-    color: #333;
-  }
+  .clear-all {
+    color: $grey-500;
 
-  span.applied-taxons {
-    color: $clr-blue;
+    &:hover {
+      color: $grey-600;
+    }
   }
 
-  span.applied-search {
-    color: $clr-brick;
-  }
+  span {
+    color: $grey-800;
+    font-style: italic;
+
+    &.applied-properties {
+      color: #333;
```

Comment:
#333 is $grey-800 and grey-800 is already the span color. My logic says this is not needed. 
If you find it's needed, it should be $grey-800.